bc_obps/registration/tests/endpoints/_contacts/test_contact_id.py

```diff
@@ -141,43 +143,50 @@ def test_industry_user_admin_update_contact_and_address(self):
 
     def test_industry_user_admin_update_contact_and_remove_address(self):
         operator = baker.make_recipe('registration.tests.utils.operator')
-        contact = baker.make_recipe('registration.tests.utils.contact', operator=operator)
+        senior_officer_role = BusinessRole.objects.get(role_name='Senior Officer')
+        contact = baker.make_recipe(
+            'registration.tests.utils.contact', operator=operator, business_role=senior_officer_role
+        )
         TestUtils.authorize_current_user_as_operator_user(self, operator)
-        # Assert that we have only one contact(to make sure we are updating the contact and not creating a new one)
-        assert Contact.objects.count() == 1
-        # Assert that we are not creating a new address
-        assert Address.objects.count() == 2
 
-        contact_data_no_address = {
-            "first_name": "John",
-            "last_name": "Doe",
-            "email": "john.doe@example.com",
-            "phone_number": "+16044011234",
-            "position_title": "Mr.Tester",
-        }
-
-        response = TestUtils.mock_put_with_auth_role(
-            self,
-            "industry_user",
-            self.content_type,
-            contact_data_no_address,
-            custom_reverse_lazy("update_contact", kwargs={"contact_id": contact.id}),
-        )
-        assert response.status_code == 200
-        assert Contact.objects.count() == 1
-        assert Address.objects.count() == 1  # One for the operator
+        # override the get method to return the senior officer role(otherwise it will return the operation representative role and the test will fail)
+        with patch('registration.models.BusinessRole.objects.get', return_value=senior_officer_role) as mock_get:
+            # Assert that we have only one contact(to make sure we are updating the contact and not creating a new one)
+            assert Contact.objects.count() == 1
+            # Assert that we are not creating a new address
```

Comment:
Suggest adding to the comment where the two addresses come from. I always forget which bakers add addresses